packages/service-worker/config/test/generator_spec.ts

```diff
@@ -523,6 +523,27 @@ describe('Generator', () => {
     });
   });
 
+  it('doesn\'t exceed concurrency limit', async () => {
+    const fileCount = 600;
+    const files = [...Array(fileCount).keys()].reduce((acc: any, _, i) => {
```

Comment:
Let's not use `any`:

```suggestion
    const files = [...Array(fileCount).keys()].reduce((acc: Record<string, string>, _, i) => {
```